src/shared/copy-plugins.js

```diff
@@ -37,17 +36,15 @@ module.exports = function runCopyPlugins (params, paths, callback) {
               if (target && isAbsolute(target)) return rej(ReferenceError(`'target' path '${target}' cannot be absolute`))
 
               // Sure what's one more nested sequence of ops?
-              series(shared.map(share => {
+              series(lambdaSrcDirs.map(lambda => {
                 return function copier (callback) {
-                  if (paths.shared[share]) {
-                    let isNode = lambdasBySrcDir[share].config.runtime.startsWith('nodejs')
-                    let filename = target || basename(source)
-                    let nodeModules = join(share, 'node_modules', filename)
-                    let vendorDir = join(share, 'vendor', filename)
-                    let dest = isNode ? nodeModules : vendorDir
-                    cp(src, dest, params, callback)
-                  }
-                  else callback()
+                  let { config, src } = lambdasBySrcDir[lambda]
```

Comment:
I believe this `src` var overwrites the previous usage on line 33. Causes the hydrate.copy copy method to hydrate the target Lmabda with itself 😬

Reply:
Right. Weird that the tests passed in a way that they should have. Idk, I can probably explain it with some time but I'll just fix it, ty for the eagle eye, and always remember to stop coding *before* your brain shuts down at the end of the day, folks!